docs/source/tutorial_torch_generator_agent.md

```diff
@@ -0,0 +1,154 @@
+# Using Torch Generator Agent
+
+**Authors**: Stephen Roller, Eric Smith
+
+`parlai.core.torch_generator_agent.TorchGeneratorAgent` is an abstract parent class that provides functionality for building autoregressive generative models. Extending `TorchGeneratorAgent` requires your model conform to a strict interface, but then provides you rich functionality like beam search and sampling.
```

Comment:
Maybe let's just use the short name, `TorchGeneratorAgent`, not the full absolute path? You make the call.